changes/pr4971.yaml

```
@@ -0,0 +1,2 @@
+fix:
```

```suggestion
enhancement:
```
This isn't really a bug and we tend to lean towards enhancements when improving usability